packages/icons/icons.json

```diff
@@ -4184,5 +4184,20 @@
         "tags": "footnote, notebook, note, reference, text, citation",
         "group": "editor",
         "codepoint": 62294
+    },
+    {
+        "displayName": "Binary",
+        "iconName": "binary",
+        "tags": "data, type, number, numeric, format",
+        "group": "miscellaneous",
+        "codepoint": 62295
+    },
+    {
+        "displayName": "Curly braces",
+        "iconName": "curly-braces",
+        "tags": "data, type, format，punctuation, grammar",
```

Comment:
can you explain the connection with "data" for this icon?

What is this icon going to be used for?

does it belong in the "editor" group of icons?

Reply:
We want to use this icon for the "Struct" data type in Pipeline Builder.